Source/Plugins/Sampling/SamplingFilters/SampleSurfaceMeshSpecifiedPoints.cpp

```diff
@@ -0,0 +1,280 @@
+/*
+ * Your License or Copyright Information can go here
+ */
+
+#include "SampleSurfaceMeshSpecifiedPoints.h"
+
+#include <QtCore/QString>
+#include <fstream>
+#include <sstream>
+
+#include "DREAM3DLib/FilterParameters/AbstractFilterParametersReader.h"
+#include "DREAM3DLib/FilterParameters/AbstractFilterParametersWriter.h"
+
+#include "DREAM3DLib/FilterParameters/FileSystemFilterParameter.h"
+#include "DREAM3DLib/FilterParameters/FileSystemFilterParameter.h"
+
+#include "Sampling/SamplingConstants.h"
+
+using namespace std;
```

Comment:
Using namespace is not allowed.  Instead use the full scope declaration where needed.